src/test/resources/tls-fips140-2.conf

```diff
@@ -0,0 +1,11 @@
+# One less ciphersuite in the list making it different than default configuration
```

Comment:
This file shouldn't be here, it is generated by the pipeline. Have you pushed it by mistake?

Reply:
Yes, let me remove it.